docs/atomic-update-container.1.md

```diff
@@ -0,0 +1,34 @@
+% ATOMIC(1) Atomic Man Pages
+% Giuseppe Scrivano
```

Comment:
Well as I said on the uninstall, I would prefer it atomic could figure this out for itself.